projects/1.19/assets/dimensional-doors/dimdoors/lang/zh_cn.json

```
@@ -0,0 +1,426 @@
+{
+  "itemGroup.dimdoors.dimensional_doors": "次元门物品",
+
+  "dimdoors.autogen_block_prefix": "%s制成的次元门",
+  "block.dimdoors.gold_door": "金门",
+  "block.dimdoors.quartz_door": "石英门",
+  "block.dimdoors.iron_dimensional_door": "铁质次元门",
+  "block.dimdoors.gold_dimensional_door": "金质次元门",
+  "block.dimdoors.quartz_dimensional_door": "私人次元门",
+  "block.dimdoors.unstable_dimensional_door": "不稳定的门",
+  "block.dimdoors.oak_dimensional_door": "橡木次元门",
+  "block.dimdoors.wood_dimensional_trapdoor": "橡木次元活板门",
+  "block.dimdoors.stone_door": "石门",
+  "block.dimdoors.dungeon_door": "地牢门",
+  "block.dimdoors.dimensional_portal": "次元传送门",
+  "block.dimdoors.black_fabric": "变化之壁",
+  "block.dimdoors.white_fabric": "白色染色变化之壁",
```

```suggestion
  "block.dimdoors.white_fabric": "白色变化之壁",
```
原文没有`Stained`，请注意全局修改